src/datetimepicker.windows.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
```

Comment:
This came up in another repo. Not sure it's kosher to put msft copyright in a community repo we don't matinain.

Reply:
Alex/Kai told me in an earlier comment to add it to all these files. Can you link the discussion you're referencing?

Reply:
They'll likely know better on this. IIRC there was one community repo that asked that copyright headers be removed. @kaiguo do you remember the story there?

Reply:
Hmm, I guess that's a @chrisglein question, should we put Microsoft copyright info in a community repo we don't own?

Reply:
Where we landed for the first couple repos (camera, video, etc.) is to leave it off. We match the pattern of the target repo. I'll check offline, but for now assume that's the case.

Reply:
Sounds good, the Android/iOS files in this repo have the MIT license from Facebook, so I'll keep ours for now.